elements/pfe-dropdown/src/pfe-dropdown-item.js

```diff
@@ -0,0 +1,79 @@
+import PFElement from "../../pfelement/dist/pfelement.js";
+
+class PfeDropdownItem extends PFElement {
+  static get tag() {
+    return "pfe-dropdown-item";
+  }
+
+  get templateUrl() {
+    return "pfe-dropdown-item.html";
+  }
+
+  get styleUrl() {
+    return "pfe-dropdown-item.scss";
+  }
+
+  static get observedAttributes() {
+    return ["pfe-type", "disabled"];
+  }
+
+  constructor() {
+    super(PfeDropdownItem);
+
+    this._container = this.shadowRoot.querySelector(`.${this.tag}__container`);
+    this._item = this.shadowRoot.querySelector("slot").assignedNodes()[1];
+  }
+
+  attributeChangedCallback(attr, oldValue, newValue) {
+    switch (attr) {
+      case "pfe-type":
+        this._setAccessibility();
+        break;
+      case "disabled":
+        this._setDisabled();
+        break;
+      default:
+        break;
+    }
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+  }
+
+  _setAccessibility() {
+    if (this._container && this._item) {
+      const type = this.getAttribute("pfe-type");
+      if (type) {
+        switch (type) {
+          case "link":
+            this._container.setAttribute("role", "none");
+            this._item.setAttribute("role", "menuitem");
+            break;
+          case "action":
+            this._container.setAttribute("role", "menuitem");
+            this._item.removeAttribute("role");
+            break;
+          case "separator":
```

Comment:
We never reach the "separator" case because the element doesn't have anything in the slot so `this._item` is empty for `pfe-type="separator"`. That's why the test is failing. We'll need to change this condition.